airflow/ui/jest.config.js

```diff
@@ -23,5 +23,6 @@
 const neutrino = require('neutrino');
 
 process.env.NODE_ENV = process.env.NODE_ENV || 'test';
+process.env.WEBSERVER_URL = process.env.WEBSERVER_URL || 'http://localhost:9999';
```

Comment:
@ashb curious to get your opinion on if we should do this or have a fully separate test environment

We just need some sort of realistic url.

Reply:
By test environment what do you mean by a separate env? Like a full running instance? 

If this is just for mocked URLs then this is fine.

Reply:
Yep. just mocks.